co-operative.md

```diff
@@ -30,48 +30,59 @@ You can contact us the following ways:
 - 🔗 [hypha.coop](https://hypha.coop)
 - 📧 hello@hypha.coop
 
-## Member-owners and Employees
+## Members and Employees
 
-Every **member-owner** is both an employee and co-owner of Hypha. There are 
-seven member-owners, all of which are founding employees, with four Directors.
+Every **member-worker**, or **member** in short, is both a member and an employee of Hypha.
+There are seven members, all of which are founding members, with six also serving as directors.
 
-A **member-owner**:
+A **member**:
   - Has completed a membership application (and/or been officially welcomed into 
     the membership)
-  - Is a *permanent* or *founding* employee
+  - Is a *founding*, or *permanent*, member if they are one of the seven members who founded Hypha
   - Is considered:
-    - *active* if they are in good standing with their membership fees and have 
-      not requested to change their status
+    - *active* if they have not requested to change their status
     - *inactive* if they have requested to change their status
       - Inactive members are unable to vote, serve as a director, participate in 
         projects, or work as an employee for so long as they remain so
       - Inactive members can request to become active at any time
 
 An **employee**: 
+  - Works under a *contract of service* with Hypha
   - Is "someone who performs work for an employer for wages," meeting the 
     [*Employment Standards Act, 2000*][es-act] [definition of employee][esa-employee] 
   - Is considered *permanent* if they:
     - Have completed a probationary period of one year 
     - Are not working under contract
 
 A **contractor**:
+  - Works as an *independent* contractor under a *contract for service* with Hypha
   - Provides services clearly specified in a time-limited contractor agreement
+  - May be a natural person or incorporated entity, domestic or foreign
 
 A **collaborator**:
   - Participates in open Hypha activities and projects without a formal employment
     relationship
 
-As a worker co-operative, there are specific conditions regarding employees and members, 
-addressed in Article [1], [1.1], [144] and [145.1] of the [*Ontario Co-operative Corporations Act*][coop-act]. 
+As every member holds a dual position as both a member and an employee of Hypha,
+they must make a distinction between *member duties* and *employee work*.
+Members are expected to acknowledge these components of their relationship with Hypha by signing the *Membership Acknowledgement*.
+
+Employees, as well as contractors and collaborators engaged in certain initiatives with Hypha,
+are expected to sign an *Employee and Contractor Intellectual Property Agreement* and comply with
+[tax obligations as applicable to them](https://handbook.hypha.coop/finance.html#gsthst-rt).
+Hypha currently does not employ anyone as *member-contractor* or *dependent contractor*.
```

Comment:
what is a "member-contractor"? (If this is saying someone cant be a contractor + a member maybe we just say like that?)

Reply:
```suggestion
```